src/patternfly/components/Accordion/accordion-button-toggle.hbs

```diff
@@ -0,0 +1,7 @@
+<span class="pf-c-accordion__button-toggle{{#if accordion__button-toggle--modifier}} {{accordion__button-toggle--modifier}}{{/if}}"
```

Comment:
Is there any reason why you can't just have the `.pf-c-accordion__button-toggle-icon` ? I'd say we can take the same approach here as we do in the dropdown where we just use an icon and not need the wrapper.

Reply:
yep you're right!